stage1/init/init.go

```diff
@@ -157,6 +157,40 @@ func withClearedCloExec(lfd int, f func() error) error {
 	return f()
 }
 
+func forwardedPorts(c *Container) ([]networking.ForwardedPort, error) {
+	fps := []networking.ForwardedPort{}
+
+	for _, ep := range c.Manifest.Ports {
+		n := ""
+		fp := networking.ForwardedPort{}
+
+		for an, a := range c.Apps {
+			for _, p := range a.App.Ports {
+				if p.Name == ep.Name {
+					if n == "" {
+						fp.Protocol = p.Protocol
+						fp.HostPort = ep.HostPort
+						fp.PodPort = p.Port
+						n = an
+					} else {
+						return nil, fmt.Errorf("Ambiguous exposed port in PodManifest: %q and %q both define port %q", n, an, p.Name)
```

Comment:
@philips The PodManifest has a global section that defines exposed ports. Each item has a name but that name is not namespaced to the ACI. Therefore there's a chance of collision. It was a conscious decision not to namespace but we can reconsider.